hyper/cli.py

```diff
@@ -93,9 +95,13 @@ def make_positional_argument(parser):
 
             search==hyper
 
-        '=' Data fields to be serialized into a JSON object:
+        '=' String data fields to be serialized into a JSON object:
 
             name=Hyper  language=Python  description='CLI HTTP client'
+
+        '@=' JSON data fields
+
+            name='{"name": "John", "surname": "Doe"}' list='[1, 2, 3]'
```

Comment:
Shouldn't it have `@=` there too, like `name@='{"name": "John", "surname": "Doe"}' list@='[1, 2, 3]'`, or am I missing something?

Reply:
You're completely right, I'll fix that. Thanks for noticing